metadata/src/main/resources/common/metadata/PartitionRecord.json

```diff
@@ -34,6 +34,8 @@
     { "name": "Leader", "type": "int32", "versions": "0+", "default": "-1",
       "about": "The lead replica, or -1 if there is no leader." },
     { "name": "LeaderEpoch", "type": "int32", "versions": "0+", "default": "-1",
-      "about": "An epoch that gets incremented each time we change the ISR." }
+      "about": "An epoch that gets incremented each time we change the ISR." },
+    { "name": "PartitionEpoch", "type": "int32", "versions": "0+", "default": "-1",
```

Comment:
Could you file a separate jira to follow up on PartitionEpoch post 2.8?

Reply:
filed KAFKA-12349